RNASeqTools_TSS/rnaseqtools_tss.jl

```diff
@@ -0,0 +1,105 @@
+# # RNASeq data analysis using RNASeqTools
+#
+# Requirements:
+#
+# * Julia 1.8.5
+# * RNASeqTools 0.3.4
+# * sra-tools 3.0.3
+# * samtools 1.6
+# * bwa-mem2 2.2.1
+#
+# In this tutorial, we will first download a few files (genome, annotation, sequencing reads) with sra-tools. 
+# We will then align the reads to the genome using bwa-mem2. We will then use the various Packages from BioJulia 
+# through the interface of RNASeqTools to load the alignments file, compute the coverage from it and do some 
+# simple signal detection in the coverage.
+
+using Pkg
+Pkg.add(url="https://github.com/maltesie/RNASeqTools#v0.3.4")
+
+using RNASeqTools
+
+# ## Downloading the data
+#
+# Our data will be from V. Cholerae, we need a genome and its annotation: First go to [NCBI](https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_013085075.1/),
+# after clicking Download, select .fasta and .gff and click Download again. Then extract the content of the archive 
+# directory /ncbi_dataset/data/GCF_013085075.1/ into the folder containing this notebook. The folder should then
+# look like:
+#
+# RNASeqTools_TSS
+#   ├── GCF_013085075.1_ASM1308507v1_genomic.fna
+#   ├── genomic.gff
+#   ├── Manifest.toml
+#   ├── Project.toml
+#   ├── rnaseqtools_tss.ipynb
+#   └── rnaseqtools_tss.jl
+
+# NCBI stores reads in its internal format SRA, we can use prefetch and fastq-dump from the sra-tools to 
+# download reads and convert them to the fastq format. The reads in sample SRR1602510 come from a TEX-treated 
+# sample and can be used to identify primary transcription start sites.
+
+download_prefetch(["SRR1602510"], @__DIR__)
+
+# ## Aligning reads with bwa-mem
+#
+# RNASeqTools provides a wrapper to bwa-mem2. We will use the genome file from the NCBI dataset downloaded earlier.
+# align_mem also calls samtools to sort and index the .bam file and creates a .log file containing a useful report
+# from samtools stats
+
+align_mem(joinpath(@__DIR__, "SRR1602510_1.fastq.gz"), joinpath(@__DIR__, "GCF_013085075.1_ASM1308507v1_genomic.fna"))
+
+# ## Annotation of alignments and exploration of the data
+#
+# To annotate the alignments, we need to read the alignments file and the annotation file, then we can 
+# explore the annotated alignments.
+
+#read features with types ["rRNA", "tRNA", "CDS"] from the .gff file nad use ["ID", "Name", "locus_tag"] parameters to name the feature
```

Comment:
```suggestion
## read features with types ["rRNA", "tRNA", "CDS"] from the .gff file nad use ["ID", "Name", "locus_tag"] parameters to name the feature
```

I prefer this `Literate.jl` syntax for actual code comments (better to have a space after `#` IMO, and this is the way to do that